src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -0,0 +1,205 @@
+package reposense.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+import java.util.logging.Logger;
+
+import reposense.system.LogsManager;
+
+/**
+ * Represents author configuration information from CSV config file for a single repository.
+ */
+public class AuthorConfiguration {
+    public static final String DEFAULT_BRANCH = "HEAD";
+    private static final Logger logger = LogsManager.getLogger(AuthorConfiguration.class);
+
+    private RepoLocation location;
+    private String branch;
+    private String displayName;
```

Comment:
Is there a need for `AuthorConfiguration` to hold the `displayName`?

Reply:
Hmmm, maybe can make this portion print the repository location + branch, since `AuthorConfiguration` doesn't needs to have a displayName